packages/m2m-gateway/src/api/tenantApiConverter.ts

```diff
@@ -0,0 +1,14 @@
+import { m2mGatewayApi, tenantApi } from "pagopa-interop-api-clients";
+
+export function toM2MTenant(tenant: tenantApi.Tenant): m2mGatewayApi.Tenant {
```

Comment:
Suggestion, for consistency with the Api converter already present for delegations 
```suggestion
export function toM2MGatewayApiTenant(tenant: tenantApi.Tenant): m2mGatewayApi.Tenant {
```